src/storage/mongodb/TagStorage.ts

```diff
@@ -284,7 +295,7 @@ export default class TagStorage {
   public static async getTags(tenantID: string,
       params: {
         issuer?: boolean; tagIDs?: string[]; userIDs?: string[]; dateFrom?: Date; dateTo?: Date;
-        withUser?: boolean; withUsersOnly?: boolean; withNbrTransactions?: boolean; search?: string, defaultTag?: boolean, active?: boolean
+        withUser?: boolean; withUsersOnly?: boolean; withNbrTransactions?: boolean; search?: string, defaultTag?: boolean, active?: boolean, visualID?: string[]
```

Comment:
The added argument naming and ordering is inconsistent: 
Please create an helper method `getTagByVisualIDs`, rename `visualID?` to `visualIDs?` and put `visualIDs?` param after `userIDs?`

And reuse `getTagByVisualIDs` in `getTagByVisualID`